sutro/sdk.py

```diff
@@ -496,94 +496,136 @@ def infer(
             str: The ID of the inference job.
 
         """
-        if isinstance(model, list) == False:
-            model_list = [model]
-            stay_attached = (
-                stay_attached if stay_attached is not None else job_priority == 0
-            )
-        else:
-            model_list = model
-            stay_attached = False
-
-        if isinstance(model_list, list):
-            if isinstance(name, list):
-                if len(name) != len(model_list):
-                    raise ValueError(
-                        "Name list must be the same length as the model list."
-                    )
-                name_list = name
-            elif isinstance(name, str):
-                raise ValueError("Name must be a list if using a list of models.")
-            elif name is None:
-                name_list = [None] * len(model_list)
-        else:
-            if isinstance(name, list):
-                raise ValueError(
-                    "Name must be a string or None if using a single model."
-                )
-            name_list = [name]
+        # Default stay_attached to True for prototyping jobs (priority 0)
+        if stay_attached is None:
+            stay_attached = job_priority == 0
+
+
+        json_schema = None
+        if output_schema:
+            # Convert BaseModel to dict if needed
+            json_schema = normalize_output_schema(output_schema)
+
+        return self._run_one_batch_inference(
```

Comment:
Shouldn't this go away with the new addition? Like infer just goes back to being the equivalent of running one batch inference?